421421412421
【发现已存在的游戏窗口】
【现在处于主菜单】
【目标出现】images/battery.png 置信度是：0.9999996423721313
Using CPU. Note: This module is much faster with a GPU.
当前电量为：58 
任务开始，执行 0 次循环
【现在处于主菜单】
【目标出现】images/battery.png 置信度是：0.9999998211860657
Using CPU. Note: This module is much faster with a GPU.
-------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 751, in run
    battery_remaining=battery_check()
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 249, in battery_check
    result=int(result[0])
               ~~~~~~^^^
IndexError: list index out of range

Exception:主线程出错
按回车键退出(Enter)Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 751, in run
    battery_remaining=battery_check()
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 249, in battery_check
    result=int(result[0])
               ~~~~~~^^^
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 763, in <module>
    run()
    ~~~^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 760, in run
    input('Exception:主线程出错\n按回车键退出(Enter)')
    ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen codecs>", line 322, in decode
KeyboardInterrupt
